ChatGPT Update

Henkjan1110 opened the pull request:
Hier de pagina geupdatet met ChatGPT. Ik vind echt dat hij de tekst hier veel beter maakt. Hij deelt het ook echt in 'hapklare brokken' op, en met de verduidelijkte uitleg erbij vind ik het hier echt een wereld van verschil!

Henkjan1110 committed:
ChatGPT Update

Hier de pagina geupdatet met ChatGPT. Ik vind echt dat hij de tekst hier veel beter maakt. Hij deelt het ook echt in 'hapklare brokken' op, en met de verduidelijkte uitleg erbij vind ik het hier echt een wereld van verschil!

nickwkoning commented on data/week5/1-learning-object-oriented-programming.md:
````diff
@@ -168,7 +180,7 @@ public class Clock {
 ```
 
-The way the program functions has become increasingly clearer. When you compare our program below to the original one that was made up of integers, you'll find that the program's readability is superior.
+This way the program functions have become increasingly clearer. When you compare our program below to the original one (made up of integers), you'll find that the program's readability is superior.
````
Engels

nickwkoning replied:
De eerste zin klopt niet (grammaticaal wel, maar de inhoud klopt niet!)

nickwkoning commented on data/week5/1-learning-object-oriented-programming.md:
````diff
@@ -179,22 +191,19 @@ while (true) {
 ```
 
-The clock we implemented above is an object whose functionality is based on "simpler" objects, i.e., its hands. This is precisely the **great idea behind ​​object-oriented programming: a program is built from small and distinct objects that work together**
+The clock we implemented earlier is a prime example of an object that relies on "simpler" objects, namely its hands, for its functionality. This is the fundamental principle of object-oriented programming: building programs from small, distinct objects that collaborate with one another.
````
Is het met 'earlier' duidelijk welk voorbeeld wordt bedoeld?

nickwkoning commented on data/week5/1-learning-object-oriented-programming.md:
```diff
@@ -179,22 +191,19 @@ while (true) {
+- `public Timer()`: creates a new timer.
+- `public String toString()`: returns a string representation of the timer in the format "seconds: hundredths of a second", where both the seconds and the hundredths of a second are represented by two numbers. For example, "19:83" would represent the time 19 seconds, 83 hundredths of a second.
+- `public void advance()`: moves the timer forward by a hundredth of a second.
+You can test the timer's functionality in the main program provided. The program prints the timer and advances it once every hundredth of a second. Once you've finished implementing the `Timer` class, please return it to the server.
```
return it to the server...? Stond er eerst ook al, maar ik snap hem niet.

nickwkoning replied:
Misschien dat ik het verkeerd lees

Henkjan1110 committed:
ChatGPT poging 2

Geprobeerd je comments te verwerken. Laat maar weten als ik nog iets moet aanpassen!

nickwkoning committed:
Update 1-learning-object-oriented-programming.md

nickwkoning committed:
Update 1-learning-object-oriented-programming.md
